resources/knative-eventing/charts/knative-eventing/templates/post-upgrade-job.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: knative-eventing-post-upgrade
+  namespace: knative-eventing
+  annotations:
+    "helm.sh/hook": "post-upgrade"
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      containers:
+      - name: post-upgrade
+        image: eu.gcr.io/kyma-project/snapshot/test/post-upgrade:test
```

Comment:
We may create a configmap attached to this job to get rid of this interim image.